### PR TITLE
Fix docs prod build

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/dev/parcel-resolver-docs/DocsResolver.js [hunk 1]
```diff
@@ -33,10 +33,11 @@ module.exports = new Resolver({
...
         // HACK: ensure source code is used to build types, not compiled code.
         // Parcel removes the source field from package.json when the code comes from node_modules.
+        // these are full filepaths, so don't check if they start with the pattern, they won't
         if ((
-          /^@(react-spectrum|react-aria|react-stately|internationalized|react-types|spectrum-icons|adobe\/react-spectrum)/g.test(resolved.filePath)
-            || /^(react-aria|react-stately)/g.test(resolved.filePath)
-          ) && resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
+          /@(react-spectrum|react-aria|react-stately|internationalized|react-types|spectrum-icons|adobe\/react-spectrum)/g.test(resolved.filePath)
+          || /(react-aria|react-stately)/g.test(resolved.filePath)
+        ) && resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
```
Comment (devongovett) [inline packages/dev/parcel-resolver-docs/DocsResolver.js:40]: nit: you could probably remove `react-types` from the regex and then you wouldn't need this extra `includes` check?

File: scripts/compareAPIs.js [hunk 2]
```diff
@@ -22,7 +22,13 @@ async function compare() {
   let branchDir = path.join(__dirname, '..', 'dist', 'branch-api');
   let publishedDir = path.join(__dirname, '..', 'dist', 'published-api');
   if (!(fs.existsSync(branchDir) && fs.existsSync(publishedDir))) {
-    console.log('one of the api directories does not exist');
+    if (!fs.existsSync(publishedDir && fs.existsSync(branchDir))) {
+      console.log('not showing private package', branchDir);
+    } else if (fs.existsSync(publishedDir && !fs.existsSync(branchDir))) {
```
Comment (solimant) [inline scripts/compareAPIs.js:27]: Are we supposed to call `existsSync` like that? Or is the closing parenthesis off?

**Review by devongovett — DISMISSED** [review]

File: scripts/compareAPIs.js (hunk 2 above)
Comment (snowystinger) [reply scripts/compareAPIs.js:27]: ack! good catch! good thing it's not a common code path

File: scripts/compareAPIs.js [hunk 3]
```diff
@@ -22,7 +22,13 @@ async function compare() {
   let branchDir = path.join(__dirname, '..', 'dist', 'branch-api');
   let publishedDir = path.join(__dirname, '..', 'dist', 'published-api');
   if (!(fs.existsSync(branchDir) && fs.existsSync(publishedDir))) {
-    console.log('one of the api directories does not exist');
+    if (!fs.existsSync(publishedDir) && fs.existsSync(branchDir)) {
+      console.log('not showing private package', branchDir);
+    } else if (fs.existsSync(publishedDir) && !fs.existsSync(branchDir)) {
+      console.log('something has probably gone wrong, the package exists published but not locally', publishedDir);
```
Comment (solimant) [inline scripts/compareAPIs.js:28]:
```suggestion
      console.log('something has probably gone wrong, the package exists in published but not locally', publishedDir);
```

**Review by solimant — APPROVED** [review]

File: scripts/compareAPIs.js (hunk 3 above)
Comment (snowystinger) [reply scripts/compareAPIs.js:28]: mine is correct, yours would need to be `the package exists in npm`

**Pull request merged**